### PR TITLE
dschinazi editorial pass on QuicTransport

### PR DESCRIPTION
**Pull request by DavidSchinazi**

### CODE REVIEW

File: draft-vvv-webtransport-quic.md [hunk 1]
```diff
@@ -349,14 +356,14 @@ malicious clients from overloading the network.
...
+QUIC being an acknowledgement-based protocol; if the client is attempting to
+send data, and the server does not send any ACK frames in response, the client
+side of the QUIC connection will time out.
 
 QuicTransport prevents WebTransport clients from connecting to arbitrary non-Web
 servers through the use of ALPN.  Unlike TLS over TCP, successful ALPN
-negotiation is mandatory in QUIC.  Thus, unless the server explicitly picks `wq`
-as the ALPN value, the TLS handshake will fail.
+negotiation is mandatory in QUIC.  Thus, unless the server explicitly picks the
+WebTransport ALPN value, the TLS handshake will fail.
```
Comment (vasilvv) [inline draft-vvv-webtransport-quic.md]: QuicTransport ALPN value

File: draft-vvv-webtransport-quic.md [hunk 2]
```diff
@@ -82,17 +82,23 @@ unidirectional streams, bidirectional streams and datagrams.
...
 to be available in web browsers and server-side web frameworks.  This makes it a
 compelling transport to base a WebTransport protocol on.
 
-This document defines QuicTransport, an adaptation of QUIC to WebTransport
-model.  The protocol is designed to be low-overhead on the server side, meaning
-that server software that already has a working QUIC implementation available
-would not require a large amount of code to implement QuicTransport.  Where
-possible, WebTransport concepts are mapped directly to the corresponding QUIC
-concepts.
+This document defines QuicTransport, an instantiation of WebTransport which is
```
Comment (vasilvv) [inline draft-vvv-webtransport-quic.md]: I actually have trouble understanding this sentence now.

Comment (DavidSchinazi) [reply draft-vvv-webtransport-quic.md]: Fixed

File: draft-vvv-webtransport-quic.md (hunk 1 above)
Comment (DavidSchinazi) [reply draft-vvv-webtransport-quic.md]: Fixed

**Pull request merged**